beacon-chain/rpc/eth/beacon/sync_committee.go

```diff
@@ -75,6 +122,28 @@ func currentCommitteeIndicesFromState(st state.BeaconState) ([]types.ValidatorIn
 	return committeeIndices, committee, nil
 }
 
+func nextCommitteeIndicesFromState(st state.BeaconState) ([]types.ValidatorIndex, *ethpbalpha.SyncCommittee, error) {
```

Comment:
I think there's already a similar helper out there, and I will double-check

Reply:
There's a similar one in the blockchain package but since the rpc package uses a helper like this for the current committee I figured it was better to mimic here